source/collaborate/chat-with-copilot.rst

```diff
@@ -90,12 +94,12 @@ Bring Copilot into any conversation
 .. include:: ../_static/badges/ent-only.rst
   :start-after: :nosearch:
 
-Invoke the power of AI by @mentioning the Copilot bot (@copilot) in any thread to bring AI's capabilities to your conversation, allowing for quick extraction of information or transformation of discussions into charts, resources, documentation, articles, and more. Copilot will find action items and open questions in new messages. With the power of Mattermost integrations and interoperability, the potential to enhance your workflow is limitless.
+Invoke the power of AI by @mentioning the Copilot bot with ``@copilot`` in any thread to bring AI's capabilities to your conversation, allowing for quick extraction of information or transformation of discussions into charts, resources, documentation, articles, and more. Copilot will find action items and open questions in new messages. With the power of Mattermost integrations and interoperability, the potential to enhance your workflow is limitless.
```

Comment:
Should we make it clear that `copilot` is the bot's username? Should we just use the default value of that field?

Reply:
Updated